execution_nodes/execute_node.js

```diff
@@ -83,64 +81,44 @@ app.post('/requestRun', function(req, res){
           "Content-Type": "application/json",
           "Content-Length": Buffer.byteLength(body)
       },
-      key : fs.readFileSync('./ssl/key.pem'),
-      cert: fs.readFileSync('./ssl/cert.pem'),
-      rejectUnauthorized:false,
+      key : fs.readFileSync("./ssl/key.pem"),
+      cert: fs.readFileSync("./ssl/cert.pem"),
+      rejectUnauthorized:false
     };
+    var request;
 
-    var request = https.request(https_request_options,function(res)
+    exec("bash cleanup.sh ".concat(submission_id + " " + lab));
+    marks.pop(); //remove last space
+    comment.pop();
+    body = scores;
+    body.submission_details.id_no = submission_id;
+    body.submission_details.commit = commit;
+    body.submission_details.marks = marks;
+    body.submission_details.comment = comment;
+    body.submission_details.Lab_No = req.body.Lab_No;
+    body.submission_details.time = req.body.time;
+    body.submission_details.status = req.body.status;
+    body.submission_details.penalty = req.body.penalty;
+    body.submission_details.socket = req.body.socket;
+    body.submission_details.log = log_b64;
+    body = JSON.stringify(body);
+
+    request = https.request(https_request_options,function(res)
```

Comment:
Space needed after comma. There are similar instances where space needs to be added. Such as in between the multiple arguments given to a function. You can figure them out. http://eslint.org/docs/rules/comma-spacing